### ISSUE TITLE
Adding "" simbols on autocomplete?

### ISSUE THREAD

**Issue by sebfindling** [angular-ui/AngularJS-Atom#61]
When I used Sublime the Angular plugin autocompleted with the ="" simbols, so you would type `<div ngi` and picking the "ng-if" suggestion would insert `<div ng-if=""` and move the cursor between the `""`

Is there a chance to add this to the plugin?

**Comment by outsideris** [comment]
Good Idea. I guess it needs autocomplete-plus( #25 ). I will try to support it.

**Comment by sebdelvalle** [comment]
any news on this?
